crates/torii/core/src/sql.rs

```diff
@@ -142,7 +142,7 @@ impl Sql {
             &mut 0,
             &mut 0,
         );
-        self.query_queue.execute_all().await?;
+        self.execute().await?;
```

Comment:
How come we are still doing an execute on all of these?

Reply:
there might be some places where the queries below the `execute()` depend on the value already being update so i didn't remove it from all places at, will verify if its strictly needed here or not.

Reply:
As we enqueue in the same order, this shouldn't impact as everything will end up being executed in the correct order (at least for now that no threads are involved).

Reply:
but there are many places where we are doing `SELECT` from the tables and using it to make some decisions on what to do further, in that case if the queries in queue got executed or not matters thats why at some places we are explicitly calling `execute`

Reply:
i tried removing it from `register_model` and it broke some other places

Reply:
since those queries are not ran its not that big of a deal, we mostly want to make sure to batch storeset and other related queries.